src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = days * 40;
+
+  if (days > 2 && days < 7) {
+    totalAmount = totalAmount - 20;
+  }
+
+  if (days > 6) {
+    totalAmount = totalAmount - 50;
+  }
```

Comment:
please create variables for `50` and `20`

Reply:
Interesting that someone cheked solution and asks to fix one mistake, and the other mentor asks to correct the mistakes in his opinion :) want to say - who's next? ))